src/api/samples.js

```diff
@@ -10,8 +10,17 @@ export async function getDetailedSample(sampleId) {
 
 /**
  * Returns all the details of the samples ids given
- * @param {Array} sampleIds Ids of the samples
+ * @param {Array} ids Ids of the samples
+ * @param {orderBy} ids Field to sort the samples eg(title or -title). ref https://github.com/AlexsLemonade/refinebio/pull/298#issue-191878045
+ * @param {number} offset
+ * @param {number} limit
  */
-export async function getAllDetailedSamples(sampleIds) {
-  return Promise.all(sampleIds.map(id => getDetailedSample(id)));
+export async function getAllDetailedSamples({ ids, orderBy, offset, limit }) {
```

Comment:
I'm curious why you choose to pass in the whole thing as an object instead of individual arguments?

Reply:
haha I commented the same here https://github.com/AlexsLemonade/refinebio-frontend/pull/51#discussion_r192488331, mostly personal preference I think. When there're more than 3-4 parameters, an object makes it easier to manage them since we don't have to worry about the order they have

Reply:
I like that reasoning 👍